azurerm/resource_arm_managed_disk.go

```diff
@@ -268,6 +298,12 @@ func resourceArmManagedDiskRead(d *schema.ResourceData, meta interface{}) error
 		if osType := props.OsType; osType != "" {
 			d.Set("os_type", string(osType))
 		}
+		if diskIOPS := props.DiskIOPSReadWrite; diskIOPS != nil {
```

Comment:
We don't need the check here as `d.Set` will do it for us:
```suggestion
		d.Set("disk_iops_read_write", props.DiskIOPSReadWrite)
```

Reply:
Fixed.